.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Since we have non-default docker-file name here: [Dockerfile-release](https://github.com/volueinsight/wapi-python/blob/development/Dockerfile-release) I am wondering whether that will work or whether we have to specify the file explicitly.

Reply:
Dockerfile-release is only used for uploading the package to pypi, e.g twine, and has nothing to do with the package itself. I think it is not relevant for the purpose we are aiming for, we could consider removing it from the dependabot file.

Reply:
Now removed